site/docs/v1/tech/lambdas/self-service-registration.adoc

```diff
@@ -0,0 +1,70 @@
+---
+layout: doc
+title: Self Service Registration Validation
```

Comment:
The other lambdas have the word `Lambda` at the end of their title.

Reply:
The majority don't, actually, but we have several that do. Can add

Reply:
okay, we should rationalize. Can you add that to all of the lambda titles, please?